src/print-dependencies.sh

```diff
@@ -6,5 +6,6 @@ srcdir="$(cd "$(dirname "$0")" && pwd)"
 arch="$(arch)"
 for x in deps vmdeps; do 
     grep -v '^#' "${srcdir}/${x}.txt"
-    grep -v '^#' "${srcdir}/${x}-${arch}.txt"
+    # Allow empty deps file, i.e. in case if arch doesn't need any special deps
+    grep -v '^#' "${srcdir}/${x}-${arch}.txt" || :
```

Comment:
I generally don't like ignoring errors...how about:
```
if [ -s "${srcdir}/${x}-${arch}.txt" ]; then
  grep -v '^#' "${srcdir}/${x}-${arch}.txt"
fi
```
?

Reply:
and we could remove the aarch64 deps file

Reply:
Right, but we'd still need a `test -f` then.

Reply:
I know this seems like a silly nitpick but in fact the [first userspace thing that runs](https://github.com/dracutdevs/dracut/) is a huge mess of shell script with inconsistent error handling that runs as root and is responsible for system bringup, which is not at all what you want for that...  Let's be strict about error handling even in trivial cases so that we don't slack off in the important ones either.

Reply:
Yeah I don't disagree :).

It really depends if the file should be present and empty, possibly with comment,(IIRC we needed/wanted the placeholders in past) or shouldn't be there at all if empty. TBH has been PITA to find what is going wrong in this case.

I will fix that, by "removing the file" way.

Reply:
The other option here is we could create a `coreos-assembler.spec` and run `yum builddep` on it - even if we don't actually build an RPM from it to start.

Reply:
I think nuking that useless file and using `test -f` explicitly makes sense. No point in having it just hang around to satisfy some code path.

> The other option here is we could create a `coreos-assembler.spec` and run `yum builddep` on it - even if we don't actually build an RPM from it to start.

That's an interesting idea, though we still need the distinction between `deps.txt` and `vmdeps.txt`.